tests/unit/usecases/test_token_api.py

```diff
@@ -48,3 +50,32 @@ def test_get_tokens(self, token_api_gateway):
         token_api_gateway.get_tokens.assert_called_once_with("New", "", "", [])
         assert result
         assert sorted(result) == sorted(obj)
+
+    def test_get_token(self, token_api_gateway):
+        obj = {
+            "hits": [
+                {
+                    "id": "00",
+                    "name": "Hathor",
+                    "symbol": "HTR",
+                    "transaction_timestamp": 1649473276,
+                    "transactions_count": 3,
+                    "sort": [
+                        "00",
+                        1649473276
+                    ],
+                    "nft": False
+                }
+            ],
+        }
+
+        token_api_gateway.get_token = MagicMock(return_value=obj)
+
+        token_api = TokenApi(token_api_gateway)
+
+        result = token_api.get_token(
```

Comment:
Should we test the API with no parameters, wrong token id, ... ?

Reply:
Added tests for the gateway on 31fb9ac052a7ef10042052359a9fb8e61d795632

A call with no parameters would fall on the token list API as the only difference between them is the path parameter `token_id`